src/tests/contract_test/wallet_test.cljs

```diff
@@ -95,3 +95,22 @@
                                main-account
                                derivation-path)]
         (assert-derived-account response)))))
+
+(defn assert-wallet-tokens
+  [tokens]
+  (let [flattened-tokens (mapcat val tokens)]
+    (is (some #(= (:symbol %) "SNT") flattened-tokens))
```

Comment:
I wonder is there anything else of importance to check in the tokens that we are dependent on.
i.e that there is some particular keys that we need

Reply:
Yah. I had the same feeling and now refactoring was also thinking about it. Maybe we just check if the tokens have been returned? Something like "response is not empty"
Opened to opinions

Reply:
oh no, I think being vague is less helpful. checking that `symbol` exists is a good thing, because that is part of the contract we are dependent on.
The best thing to do here would be too look at how we are using this rpc request and then check what parts of the data we need, then we know what we need to check in these contracts so that Status Go avoids making breaking changes for this endpoint 👍

Reply:
@ilmotta any toughts?

Reply:
Nothing to add really to Jamie's comment above. I think that's a good description of the goal of all contract tests.